airbyte-integrations/connectors/source-mailchimp/source_mailchimp/source.py

```diff
@@ -93,9 +105,15 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             Automations(authenticator=authenticator),
             Campaigns(authenticator=authenticator),
             EmailActivity(authenticator=authenticator, campaign_id=campaign_id),
+            InterestCategories(authenticator=authenticator, parent=Lists(authenticator=authenticator)),
```

Comment:
Would it be better to avoid creating multiple objects of the same stream?
```
        lists = Lists(authenticator=authenticator)
        interest_categories = InterestCategories(authenticator=authenticator, parent=lists)
        return [
            Automations(authenticator=authenticator),
            Campaigns(authenticator=authenticator),
            EmailActivity(authenticator=authenticator, campaign_id=campaign_id),
            interest_categories,
            Interests(
                authenticator=authenticator,
                parent=interest_categories,
            ),
            lists,
            ListMembers(authenticator=authenticator),
            Reports(authenticator=authenticator),
            Segments(authenticator=authenticator),
            Tags(authenticator=authenticator, parent=lists),
            Unsubscribes(authenticator=authenticator, campaign_id=campaign_id),
        ]
```

Reply:
Good call! I've made this change as suggested